packages/aws-cdk-lib/aws-ecs/lib/container-definition.ts

```diff
@@ -251,6 +252,14 @@ export interface ContainerDefinitionOptions {
    */
   readonly hostname?: string;
 
+  /**
+   * When this parameter is true, you can deploy containerized applications that require stdin or a tty to be allocated.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinition.html#cfn-ecs-taskdefinition-containerdefinition-interactive
+   * @default - false
+   */
+  readonly interactive?: boolean | IResolvable;
```

Comment:
```suggestion
  readonly interactive?: boolean;
```
The property should be resolved.